### ISSUE TITLE
setterm: cannot (un)set powersave mode: Inappropriate ioctl for device

### ISSUE THREAD

**Issue by Blue-C-Druidus** [substring/os#34]
Hello. I recently updated my GA installation, and the next time I boot my PC this message  displayed: 
```setterm: cannot (un)set powersave mode: Inappropriate ioctl for device```
Afterwards, the screen doesn't display anything on my CRT, and on an LCD it says "input not supported" (obviously because it doesn't support 15kHz, but it would display the GA menu before iirc).

**Comment by substring** [comment]
Hi!

That setterm message is just a warning that appeared with a recent kernel update. Now for the 15kHz error, no one reported that with an update so far. What happened at worst is the terminal not showing your typing. This can be solved with `stty echo`;

What GPU are you using ?

**Comment by Blue-C-Druidus** [comment]
It's an XFX Radeon HD5450. Not sure if it helps, but I also have Retroarch as my Frontend.

**Comment by substring** [comment]
Do you mean you don't even see gasetup when exiting RA ?

**Comment by Blue-C-Druidus** [comment]
Yeah, I tried exiting RA blind but I didn't have any response. I could try again though.

**Comment by Blue-C-Druidus** [comment]
> Do you mean you don't even see gasetup when exiting RA ?

Ok tried again, I was able to display gasetup on the LCD

**Comment by Blue-C-Druidus** [comment]
Here you go:
https://transfer.sh/eyM9GLj0Nf/ga-support.tar.zst

**Comment by substring** [comment]
Thanks, I'll check that in a few hours

**Comment by Blue-C-Druidus** [comment]
> have you changed anything to RA verbosity ? it's missing most of the usual retroarch log info + loads of errors of switchres

I don't recall editing any such options in RA (at least not intentionally). I was also wondering if my switchres setup was wrong. While while the games displayed just fine, the gasetup screen on CRT was squished.

**Comment by Blue-C-Druidus** [comment]
I changed the verbosity to true through the cfg file and amde a new link. Hope this helps.
https://transfer.sh/7DP7jJ8J62/ga-support.tar.zst

**Comment by substring** [comment]
Not seeing anything better + I don't see anything happening on VGA-1. Is your CRT disconnected ?

I need you to start a rom, I'm not sure you launched kingdmgp after changing RA's log verbosity, because the log is still missing much content. But ... I think I get it: having RA set as a frontend probably doesn't  redirect the log to where I need it.

Ok, have you tried switching back to AM as a frontend and see if the problem still occurs ?

**Comment by substring** [comment]
nothing still. Does the same happen if you switch from KMS to X ?

**Comment by Blue-C-Druidus** [comment]
> nothing still. Does the same happen if you switch from KMS to X ?

That's so weird :/
I was certain I took care of everything.

I'm not sure what you mean by KMS and X?

**Comment by substring** [comment]
Gasetup > setup> video setup > X/KMS

**Comment by Blue-C-Druidus** [comment]
> Gasetup > setup> video setup > X/KMS

Huh, it worked but the image is squished.![20230818_193930.jpg](https://github.com/substring/os/assets/83394669/dbd73bef-341d-4935-bdf7-e3ed30a47046)

Also gasetup only seems to display on the LCD now.

**Comment by substring** [comment]
You can't always have both lcd and crt connected, this leads to inconsistency
Gasetup is launched on the terminal. Switching KMS/X doesn't affect it

**Comment by Blue-C-Druidus** [comment]
No, I mean it won't display on CRT even when it's connected alone.

Anyway thanks for the help (and thanks for your patience!), I'll mark this as solved.
